api/internal/utils/consts/api_error.go

```diff
@@ -27,7 +29,21 @@ func ErrorWrapper(handle WrapperHandle) gin.HandlerFunc {
 	return func(c *gin.Context) {
 		data, err := handle(c)
 		if err != nil {
-			apiError := err.(*ApiError)
+			apiError, ok := err.(*ApiError)
+			if !ok {
+				errMsg := err.Error()
+				if strings.Contains(errMsg, "required") ||
```

Comment:
Why not encapsulating these logics into a separate method in `ApiError`, like `ApiError.IsInvalidParam`?

Reply:
I had seen the same code in `hadler.go`.
https://github.com/apache/apisix-dashboard/blob/cb1377cf9859392ee81ee60b3d4e623b1522be1e/api/internal/handler/handler.go#L55-L61

So I think we'd better discuss about the response error.

Reply:
@starsz  sure, we need a PR to optimize it.